shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -399,6 +400,7 @@ - (instancetype)initWithBridge:(fml::WeakPtr<flutter::AccessibilityBridgeIos>)br
     _bridge = bridge;
     _uid = uid;
     _children = [[NSMutableArray alloc] init];
+    _inDealloc = NO;
```

Comment:
Don't explicitly initialize things to their defaults values, per ObjC style guide. This line can just be removed.

Reply:
```suggestion
```